pkg/cri/server/helpers_linux.go

```diff
@@ -141,8 +140,14 @@ func checkSelinuxLevel(level string) error {
 	return nil
 }
 
+// apparmorEnabled returns true if apparmor is enabled, supported by the host,
+// and if apparmor_parser is installed, and if we're not running docker-in-docker..
 func (c *criService) apparmorEnabled() bool {
-	return runcapparmor.IsEnabled() && !c.config.DisableApparmor
+	// see https://github.com/docker/docker/commit/de191e86321f7d3136ff42ff75826b8107399497
```

Comment:
This is from 2014... Do we really need it?

Reply:
Apparently we do https://github.com/opencontainers/runc/pull/2554#issuecomment-724611148